### ISSUE TITLE
Clear Old Find and Replace Entries 

### ISSUE THREAD

**Issue by gje627** [Sigil-Ebook/Sigil#87]
Love Sigil and thanks for all your hard work. One request. I used Sigil's Find and Replace frequently both in Normal and Regex modes. The problem is that my prior Find and Replace searches remain in both the Find and Replace dropdown boxes into perpetuity. Occasionally this causes a problem when I mistakenly click an old find/replace option because unlike most functions these mistakes cannot be undone. Therefore, I have two complementary requests:

1. Allow users to "clear out" old searches in both the find and replace dropdown boxes.
2. Allow mistakes made in find and replace to be "undone."

**Comment by gje627** [comment]
ADDENDUM: I figured out an easy workaround to my request #1 above. I'm on Windows 7 so I simply went to my temp folder at C:\Users\Myname\AppData\Local\sigil-ebook\sigil and edited the sigil.ini file in Notepad++ by deleting all my old searches, finds, and replaces in the following three sections:

clipboardringhistory=
find_strings=
replace_strings=

This was easy and worked perfectly so no worries about request #1.

Also, request #2 is not a huge deal so this can probably be ignored too.

Therefore, feel free to close this ticket if you would like.

Thanks again!!
